### PR TITLE
Services page

### PR DESCRIPTION
**Pull request by fares98**
Create Services page

### CODE REVIEW

File: client/src/components/Pages/Services/index.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    redirect: false,
+  };
+
+  async componentDidMount() {
+    try {
+      const services = await axios.get('api/v1/services');
+      const result = services.data.data;
+      this.setState({ servicesData: result });
+    } catch (err) {
+      console.log(err);
```
Comment (AbdallahAmmar96) [inline client/src/components/Pages/Services/index.js]: We don't handle the error by console.log

File: client/src/components/Pages/Services/index.js [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    }
+    return null;
+  };
+
+  render() {
+    const { servicesData } = this.state;
+    return (
+      <section>
+        {this.renderRedirect()}
+        <SearchInput name="search" placeholder="Search" />
```
Comment (AbdallahAmmar96) [inline client/src/components/Pages/Services/index.js]: Where is the handle for the search input

File: client/src/components/Pages/Services/index.js [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+      const services = await axios.get('api/v1/services');
+      const result = services.data.data;
+      this.setState({ servicesData: result });
+    } catch (err) {
+      console.log(err);
+    }
+  }
+
+  autoComplete = e => {
+    const enterdLetters = e.target.value;
```
Comment (justinwiley) [inline client/src/components/Pages/Services/index.js]: enterdLetters -> enteredLetters

File: client/package.json [hunk 4]
```diff
@@ -43,5 +43,6 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
-  }
-}
+  },
+  "proxy": "http://localhost:5000"
+}
```
Comment (mossa-Sammer) [inline client/package.json]: add a blank line at the end of the file

File: client/src/components/Pages/Services/index.js [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+    resultAutoComplete: '',
+  };
+
+  async componentDidMount() {
+    try {
+      const services = await axios.get('api/v1/services');
+      const result = services.data.data;
+      this.setState({ servicesData: result });
+    } catch (err) {
+      console.log(err);
```
Comment (mossa-Sammer) [inline client/src/components/Pages/Services/index.js]: console log ??

**Pull request merged**